lib/utils/duration.go

```diff
@@ -0,0 +1,54 @@
+/*
+ * Teleport
+ * Copyright (C) 2025  Gravitational, Inc.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+package utils
+
+import (
+	"encoding/json"
+	"time"
+)
+
+// Duration is used to implement JSON marshal/unmarshal for time.Duration
+type Duration time.Duration
```

Comment:
What is this used for? I don't see any reference in this PR.

Reply:
First usage is here for CockroachDB: https://github.com/gravitational/teleport.e/pull/7710

Will also use it for the other pg compatible backend client configs.